xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-api/src/main/resources/META-INF/components.txt

```diff
@@ -6,3 +6,4 @@ org.xwiki.security.authorization.DefaultAuthorizationManager
 org.xwiki.security.authorization.internal.DefaultAuthorizationManagerConfiguration
 org.xwiki.security.authorization.internal.DefaultAuthorizationSettler
 org.xwiki.security.authorization.internal.PrioritizingAuthorizationSettler
+org.xwiki.security.query.internal.DefaultQueryConfiguration
```

Comment:
As a best practice, we tend to put a newline at the end of a components.txt.

Reply:
Yes, and this should actually fail the build normally.

Reply:
I don't think we have a rule for it right now. Or maybe you suggest that we add one (which would indeed be a good idea) ?

Reply:
I'm pretty sure we have one (created by @surli AFAIR).

Reply:
Found the jira issue: http://jira.xwiki.org/browse/XCOMMONS-2119

Reply:
> Found the jira issue: http://jira.xwiki.org/browse/XCOMMONS-2119

You did not get any error when building with -Pquality @aemi-dev ?

Reply:
Actually code displayed by IntelliJ contains that newline and I remember having to add it to avoid a build failure. So I'm trying to understand why IntelliJ is displaying it and not GitHub

Reply:
In https://github.com/xwiki/xwiki-platform/blob/2cb48c6709f208b978d5ee1ddd262aa094a97e9d/xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-script/src/main/resources/META-INF/components.txt, there's a new line on my side but not on GitHub, I suppose GitHub decided to not display these trailing empty newline anymore

Reply:
As I just said, if you have a look to other `components.txt` on `xwiki-platform`, these newline are not displayed

Reply:
> I suppose GitHub decided to not display these trailing empty newline anymore

OK, strange.